### PR TITLE
[Feature][Ted] features section

### PR DESCRIPTION
**Pull request by caspted**
Screenshot 2023-07-02 at 1 56 40 AM
Screenshot 2023-07-02 at 12 18 33 AM

https://app.clickup.com/t/865cm29km

### CODE REVIEW

**Review by jedmamosto — COMMENTED** [review]
for colors (especially text colors) refer to tailwind config to follow consistency

`text-black` should be `text-dark`

`text-indigo-900` should be `text-primary-400`

etc.

**Comment by jedmamosto** [comment]
desktop images are blurred hmmmm... too compressed? 

mockup images: https://drive.google.com/drive/folders/1PfbRvNEPRDAg_HNhNJsgBnbR-pDNwXJ-?usp=sharing

get it from these lang siguro for higher qual but compress if pwede pa (without qual suffering)

**Comment by jedmamosto** [comment]
letter spacing ('featured') and font sizes are also off sa desktop screen

ORRR for the featured part just add a font weight (it has font weight of medium)

refer to Figma, e.g.: (should be lg: text-2xl)

the font size from the tablet screen (sm) is the same with the desktop



image

**Comment by jedmamosto** [comment]
the line under feature has opacity and alsooo idk if its just me pero the line seems imbalanced - like its more lengthy on the left side HAHAH

**Comment by jedmamosto** [comment]
just check sa Figma lang siguro ted for most of the changes ur gonna do (look at High Fidelity from the left part of the page)

https://www.figma.com/file/UfaSbaNNbPhbDXz1Wz15mw/UniHub-Landing-Page-02%2F06%2F2023?type=design&node-id=152-63&mode=design&t=UCHri4tFUO9Bs01n-0

other than that, gj!!

File: app/Features.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import Image from "next/image";
+
+export default function Features() {
+  return (
+    <section>
+      {/* This is for the Features header */}
```
Comment (ezerssss) [inline app/Features.tsx]: remove all comments

code should be self describing

File: app/Features.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      <div>
+        <div className="flex flex-col items-center justify-center mt-15 lg:mt-16">
+          <div>
+            <p className="w-44 font-sans text-xl text-center text-black tracking-widest sm:text-4xl lg:text-5xl">Features</p>
+          </div>
+          <div className="bg-[#191970] w-44 h-1 sm:w-96 lg:w-5/12"></div>
+        </div>
+      </div>
+
+      {/* This is for the Features images and content */}
```
Comment (ezerssss) [inline app/Features.tsx]: remove

File: app/Features.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+export default function Features() {
+  return (
+    <section>
+      {/* This is for the Features header */}
+      <div>
+        <div className="flex flex-col items-center justify-center mt-15 lg:mt-16">
+          <div>
+            <p className="w-44 font-sans text-xl text-center text-black tracking-widest sm:text-4xl lg:text-5xl">Features</p>
+          </div>
+          <div className="bg-[#191970] w-44 h-1 sm:w-96 lg:w-5/12"></div>
```
Comment (ezerssss) [inline app/Features.tsx]: dont use magic numbers or magic strings 

refer:
image


to resolve put this in the tailwind config just how we have `bg-primary-100` you can do that for this color as well

File: app/Features.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+import Image from "next/image";
+
+export default function Features() {
+  return (
+    <section>
+      {/* This is for the Features header */}
+      <div>
+        <div className="flex flex-col items-center justify-center mt-15 lg:mt-16">
+          <div>
+            <p className="w-44 font-sans text-xl text-center text-black tracking-widest sm:text-4xl lg:text-5xl">Features</p>
```
Comment (ezerssss) [inline app/Features.tsx]: can you remove `text-black` here? if yes remove it so that we can simplify the code

File: app/Features.tsx (hunk 3 above)
Comment (ezerssss) [reply app/Features.tsx]: instead of a `div` use an `hr` instead

File: app/Features.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+          </div>
+          <div className="bg-[#191970] w-44 h-1 sm:w-96 lg:w-5/12"></div>
+        </div>
+      </div>
+
+      {/* This is for the Features images and content */}
+    <div className="lg:flex lg:items-aligned lg:justify-evenly lg:mt-16">
+      <div className="flex items-aligned justify-evenly mt-7 sm:mt-14 lg:flex-col lg:justify-center ">
+        <div>
```
Comment (ezerssss) [inline app/Features.tsx]: can you remove this `div`? just have the `Image` component here, simplify the code as much as possible

File: app/Features.tsx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+            src="/mockup2.png"
+            width={111.51} 
+          /> 
+        </div>
+        <div className="custom-width-157 self-center"> 
+          <p className="text-indigo-900 text-xs text-center items-aligned justify-center sm:text-2xl lg:mt-10 lg:text-3xl lg:my-3">Meetups made easy</p> 
+        </div>
+      </div>
+      <div className="flex flex-row-reverse items-aligned justify-evenly mt-14 sm:mt-11 lg:flex-col lg:justify-center">
+        <div>
```
Comment (ezerssss) [inline app/Features.tsx]: remove if possible

File: app/Features.tsx [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+        <div> 
+          <Image 
+            alt="Mockup 2" 
+            className="w-28 sm:w-56 lg:w-64 lg:items-aligned lg:ml-3" 
+            height={226} 
+            src="/mockup2.png"
+            width={111.51} 
+          /> 
+        </div>
+        <div className="custom-width-157 self-center">
```
Comment (ezerssss) [inline app/Features.tsx]: what is `custom-width-157`? doesnt look like its a tailwind CSS class name, its not even on the `globals.css`.

remove if not used

File: app/Features.tsx [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+        <div> 
+          <Image 
+            alt="Mockup 3" 
+            className="w-28 sm:w-56 lg:w-64 lg:items-aligned lg:ml-2" 
+            height={226} 
+            src="/mockup3.png"
+            width={111.51}
+          /> 
+        </div>
+        <div className="custom-width-157 self-center">
```
Comment (ezerssss) [inline app/Features.tsx]: `custom-width-157` ?

File: app/Features.tsx [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+            src="/mockup3.png"
+            width={111.51}
+          /> 
+        </div>
+        <div className="custom-width-157 self-center"> 
+          <p className="text-indigo-900 text-xs text-center items-aligned justify-center sm:text-2xl lg:mt-10 lg:text-3xl">Selling made simple</p> 
+        </div>
+      </div>
+      <div className="flex items-aligned justify-evenly mt-14 sm:mt-11 lg:flex-col lg:justify-center">
+        <div className="-ml-2">
```
Comment (ezerssss) [inline app/Features.tsx]: remove if possible

File: app/Features.tsx [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+        <div className="-ml-2"> 
+          <Image 
+            alt="Mockup 4" 
+            className="w-28 sm:w-56 lg:w-64 lg:items-aligned lg:ml-2" 
+            height={557} 
+            src="/mockup4.png"
+            width={275}
+          /> 
+        </div>
+        <div className="custom-width-157 self-center">
```
Comment (ezerssss) [inline app/Features.tsx]: `custom-width-157`

**Review by ezerssss — CHANGES_REQUESTED** [review]
please install prettier properly, or set that your editor formats on save.

refer: https://www.alphr.com/use-prettier-vs-code/#:~:text=HOME,Automatically%20Format%20Document%20on%20VS%20Code%20Using%20Prettier

**Comment by ezerssss** [comment]
> the line under feature has opacity and alsooo idk if its just me pero the line seems imbalanced - like its more lengthy on the left side HAHAH

it should be centered man, since he used `items-center justify-center` and no margins on the x-axis as well, maybe it just looks off pero it should be centered

File: app/Features.tsx (hunk 7 above)
Comment (jedmamosto) [reply app/Features.tsx]: use `w-40` nalang guro @caspted - when designin using figma use the closest value in tailwind lang siguro

**Review by ezerssss — APPROVED** [review]
did you `npm install` properly? `prettier-plugin-tailwindcss` is not applied in ur pr

maybe go to the branch `main` do 
- `git pull` 
- `npm install`
- go back to your branch `feature/features`
- `git rebase main`

and check if the `prettier-plugin-tailwindcss` is working, it should automatically sort your tailwind classNames.

then just push to this branch then merge.

Thanks!

**Review by jedmamosto — COMMENTED** [review]
remove `tracking-widest` lang guro? it looks weird

**Pull request merged**